coderd/oauth2.go

```diff
@@ -13,11 +13,11 @@ import (
 	"github.com/coder/coder/v2/coderd/database/dbtime"
 	"github.com/coder/coder/v2/coderd/httpapi"
 	"github.com/coder/coder/v2/coderd/httpmw"
+	"github.com/coder/coder/v2/coderd/identityprovider"
 	"github.com/coder/coder/v2/codersdk"
-	"github.com/coder/coder/v2/enterprise/coderd/identityprovider"
 )
 
-func (api *API) oAuth2ProviderMiddleware(next http.Handler) http.Handler {
+func (*API) oAuth2ProviderMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
 		if !buildinfo.IsDev() {
```

Comment:
This still true?

Reply:
For now, yup.  We have not actually integrated it into Backstage yet, once we have done that and verified everything works well then we will remove the dev check.